CHANGELOG.md

```diff
@@ -5,7 +5,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- Favicons are updated to reflect their provider, on a branded domain or on a /preprints/provider page
```

Comment:
I don't think we should delete this

Reply:
This line was the result of a dirty merge conflict resolution and shouldn't be here. It's already included under the Added section below. Can either delete it here or I will clean it up when I release.